BlueprintUI/Sources/Layout/EqualStack.swift

```diff
@@ -0,0 +1,118 @@
+/// An element that sizes all of its children equally, stacking them according to
+/// the specified `direction` and spacing them according to the specified `spacing`.
+///
+/// - Note: A stack is measured to accommodate its largest child in each axis.
```

Comment:
This could be a little clearer — the measurement is the size required for *all* children to be sized to match the largest child, right?

Reply:
agreed. will clarify!